code/modules/jobs/job_types/quartermaster.dm

```diff
@@ -27,6 +27,10 @@
 
 	family_heirlooms = list(/obj/item/stamp, /obj/item/stamp/denied)
 
+	mail_goodies = list(
+		/obj/item/circuitboard/machine/emitter = 3
```

Comment:
Doesn't this become a 100% chance to give the QM an emitter board?

Reply:
Nope, if a job has `exclusive mail goodies` enabled (Which most don't!) then you would be right.
Because the quartermaster has a weight of 3 on the emitter, it's rare when compared to the 56 weight on generic goodies.